src/modules/compliance/src/lib/procedures/EnsureMountPointExists.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <CommonUtils.h>
+#include <Evaluator.h>
+#include <Regex.h>
+#include <iostream>
+#include <string>
+
+namespace compliance
+{
+
+AUDIT_FN(EnsureMountPointExists, "mountPoint:Mount point to check:M")
+{
+    auto it = args.find("mountPoint");
+    if (it == args.end())
+    {
+        return Error("No mount point provided");
+    }
+    auto mountPoint = std::move(it->second);
+
+    Result<std::string> findMntOutput = context.ExecuteCommand("findmnt -knl");
```

Comment:
`findmnt <mountpoint>` will give you the information for a specific mountpoint directly.

Reply:
It'll fail when mountpoint is not a mountpoint, doing it this way gives clear distinction between findmnt failure and "not a mountpoint".

Reply:
are you saying you expect `findmnt <mountpoint>` to fail for other reasons?

Reply:
The simplest scenario for that would be findmnt missing from the system. We currently return an error in this case